CMakeLists.txt

```diff
@@ -189,7 +190,7 @@ ELSE()
     "ALL"
     STRING
     "A list of components to enable in testing and building"
-    VALID_ENTRIES BATCHED BLAS GRAPH SPARSE ALL
+    VALID_ENTRIES BATCHED BLAS LAPACK CUSOLVER ROCSOLVER GRAPH SPARSE ALL
```

Comment:
CUSOLVER and ROCSOLVER are not components they are TPLs, please remove them from this list

Reply:
Agree. Corrected. Thanks,